packages/components/src/components/navigation-item/navigation-item.stories.ts

```diff
@@ -462,3 +462,39 @@ export const Mouseless = {
     el?.shadowRoot?.querySelector('button')?.focus();
   }
 };
+
```

Comment:
1. Move this before `mouseless`
2. Add an example to `mouseless`, too.
3. Add some documentation above the story.

Reply:
Please check if documentation can be improved